app/src/lib.rs

```diff
@@ -230,6 +233,36 @@ pub struct PortalApp {
     relay_pool: Arc<RelayPool>,
     runtime: Arc<BindingsRuntime>,
 }
+#[derive(uniffi::Object, Debug)]
```

Comment:
You should derive `uniffi::Record` rather than `Object`: records are accessible on the javascript side as if they were structures, so you can see the fields. Objects are opaque, you would have to add getter methods to read the fields from js.